scripts/run_chain_tests.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env python3
+
+import os
+import psutil
+import shutil
+import signal
+import subprocess
+import time
+
+def run_process(args):
+    return subprocess.check_output(args, stderr=subprocess.STDOUT,
+                                   encoding="utf-8")
+
+def run_chain_test(chain):
+    run_process(["yarn"])
+
+    if chain == "polkadot":
+        url = "wss://rpc.polkadot.io"
```

Comment:
Would be cool if we had our own nodes - but I supposed you could not check the addresses into source control.

Reply:
I think we could input the URLs as either `CLI args` or set `env variables` for them instead.

Reply:
Yea I agree having our own nodes would be very nice. Would be a nice addition for the tools team in general.